src/index.tsx

```diff
@@ -25,20 +25,20 @@ export {
   propNames,
   propAliases,
   propEnhancers
-} from './enhancers/index'
+} from "./enhancers/index";
 
-export const hydrate = cache.hydrate
+export const hydrate = cache.hydrate;
 
 export function extractStyles() {
   const output = {
     cache: cache.entries(),
     styles: styles.getAll()
-  }
-  clearStyles()
-  return output
+  };
+  clearStyles();
+  return output;
 }
 
 export function clearStyles() {
-  cache.clear()
-  styles.clear()
+  cache.clear();
+  styles.clear();
```

Comment:
:) haha yeah let's go ahead and add prettier (and try to keep it in line with the current style of the lib if possible).